packages/sui-mono/bin/sui-mono-commit-all.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+/* eslint no-console:0 */
+const program = require('commander')
+const path = require('path')
+const { exec } = require('child_process')
+const config = require('../src/config')
+const { getSpawnPromise, showError } = require('@s-ui/helpers/cli')
+
+program
+.usage('-m "My commit message"')
+.option('-m, --message "<message>"', 'Commit message')
+.option('-t, --type "<commit-type>"', 'Commit type')
+  .on('--help', () => {
+    console.log('  Description:')
+    console.log('')
+    console.log('    Do same commit for all packages')
+    console.log('')
+    console.log('    Usefull for cross-pacakge refactors.')
+    console.log('')
+    console.log('  Examples:')
+    console.log('')
+    console.log('    $ sui-mono commit-all --type=fix -m "bump dependency to major version"')
+    console.log('    $ sui-mono --help')
+    console.log('    $ sui-mono -h')
+    console.log('')
+  })
+  .parse(process.argv)
+
+// Check mandatory parameters
+const {message, type} = program
+!message && showError('Commit message is mandatory')
+!type && showError('Commit type is mandatory')
+
+const packagesDir = path.join(process.cwd(), config.getPackagesFolder())
+
+/**
+ * Checks if given path has changes
+ * @param  {String}  path Folder to check
+ * @return {Promise<Boolean>}
+ */
+const hasChangedFiles = (path) => {
+  return new Promise((resolve, reject) => {
+    exec(`git add . && git status ${path}`, {cwd: path}, (err, output) => {
```

Comment:
I think you should at least advice that this command is going to commit everything

Reply:
it's a specific command ti commit all... I don't get your point.
Plus, it's a local thing, a "git reset" is easy. There is no push, publish or anything with side effect.
What I can do, is document it more to explain more clearly what it does. But I really don't think it's necessary.